styleguide/_themes/derek/scss/components/mainnav.scss

```diff
@@ -268,6 +270,10 @@
 
     .navbar-topItem {
       font-size: 1.07rem;
+
+      .navbar-dropDownLink {
```

Comment:
As much as I hate this nested like this, I've confirmed its necessary due to weird bootstrap specificity.